Plugins/DotNET/DotNETExports.cpp

```diff
@@ -406,10 +403,10 @@ static int32_t ClosureDelayCommand(uint32_t oid, float duration, uint64_t eventI
 {
     if (Utils::GetGameObject(oid))
     {
-        int32_t days = Globals::AppManager()->m_pServerExoApp->GetWorldTimer()->GetCalendarDayFromSeconds(duration);
-        int32_t time = Globals::AppManager()->m_pServerExoApp->GetWorldTimer()->GetTimeOfDayFromSeconds(duration);
+        uint32_t days = Globals::AppManager()->m_pServerExoApp->GetWorldTimer()->GetCalendarDayFromSeconds(duration);
+        uint32_t time = Globals::AppManager()->m_pServerExoApp->GetWorldTimer()->GetTimeOfDayFromSeconds(duration);
```

Comment:
Changed these to unsigned, as `ai->AddEventDeltaTime()` was already implicitly casting.